packages/next/export/worker.ts

```diff
@@ -344,7 +345,7 @@ export default async function exportPage({
         }
 
         if (typeof components.Component === 'string') {
-          html = components.Component
+          html = resultFromChunks([components.Component])
```

Comment:
Do you want to rename the return value to sth like `result` here since `resultFromChunks` return a `RenderResult` type? non-blocking.